modules/ROOT/nav.adoc

```diff
@@ -126,19 +126,13 @@
 *** xref:editor-plugin-version.adoc[Specify editor version & plugins]
 *** xref:plugin-editor-version-compatibility.adoc[Version compatibility reference]
 *** xref:cloud-troubleshooting.adoc[Cloud Troubleshooting]
-** Premium server-side services guide
-*** Server-side component installation with Docker
-**** Premium self-hosted bundle users
-***** xref:bundle-intro-setup.adoc[Introduction and initial setup]
-***** xref:bundle-hyperlinking-container.adoc[Hyperlinking service]
-***** xref:bundle-spelling-container.adoc[Spelling service]
-***** xref:bundle-imageproxy-container.adoc[Image Proxy service]
-**** Individually licensed plugin services
-***** xref:individual-hyperlinking-container.adoc[Hyperlinking service]
-***** xref:individual-image-proxy-container.adoc[Image Proxy service]
-***** xref:individual-spelling-container.adoc[Spelling service]
-***** xref:individual-export-to-pdf-on-premises.adoc[Export to PDF]
-***** xref:individual-import-from-word-and-export-to-word-on-premises.adoc[Import from Word and Export to Word]
+** Premium server-side services guides
+*** xref:bundle-intro-setup.adoc[Bundle intro setup]
```

Comment:
```suggestion
*** xref:bundle-intro-setup.adoc[Introduction]
```

Reply:
resolved in [b4a4ea7](https://github.com/tinymce/tinymce-docs/pull/3652/commits/b4a4ea707a0b5ac3f7c996465ea0f40b581ea568)